tools/cdb2sql/cdb2sql.c

```diff
@@ -694,7 +695,9 @@ static void process_line(char *sql, int ntypes, int *types)
     sqlstr[len] = '\0';
 
     int start_time_ms, run_time_ms;
+    gbl_in_stmt = 1;
```

Comment:
```
Imagine doing ctrl_c several times in a row--you would talk to the server trying to cancel something without good reason.
```
That's not right. The first Ctrl-C should kill the query and the next one should kill the client.